doc/tutorials/sagemaker/sme_init_ui.ipynb

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
<p>I think, here would be an enum for the keys also useful </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>